src/main/resources/db/changelog/changes/04-create-role-table.yaml

```diff
@@ -0,0 +1,20 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-role-table
+      author: vik
+      changes:
+        - createTable:
+            tableName: roles
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: name
+                  type: enum('ADMIN', 'USER')
```

Comment:
hardcode is evil)

Reply:
discussed on QnA, this variant could also be